Removed referral functions

DragonKnight1230 opened the pull request:
removed most of the referral function, except purchaseTokensERC20WithReferrer on line 333

DragonKnight1230 committed:
Removed referral functions

removed most of the referral function, except purchaseTokensERC20WithReferrer on line 333

TSavo commented on DivvyUp.sol:
```diff
@@ -370,31 +330,31 @@ contract DivvyUp is ERC20Interface, Owned, DivvyUpInterface {
      * Converts all incoming counter to tokens for the caller
      */
-    function purchaseTokensERC20WithReferrer(uint256 amount, address referrer)
+    function purchaseTokensERC20WithReferrer(uint256 amount, address)
```
This function and the one above need to be merged to be one function which does't take an address.

TSavo commented on DivvyUp.sol:
```diff
@@ -784,45 +722,28 @@ contract DivvyUp is ERC20Interface, Owned, DivvyUpInterface {
     =            INTERNAL FUNCTIONS            =
     ==========================================*/
-    function purchaseTokens(uint256 incomingCounter, address referredBy)
+    function purchaseTokens(uint256 incomingCounter, address)
```
The address needs to be removed from this function. There's no address to pass in.

DragonKnight1230 committed:
remove address

removed address in purchaseTokens function and merged purchaseTokensERC20WithReferrer to purchaseTokensERC20 function
